### PR TITLE
improve: SQL examples

### PR DESCRIPTION
**Pull request by akshayka**
Create a collection of bite-sized examples showing how to use SQL:

- querying dataframes
- parametrizing queries
- read from CSV
- read from JSON
- read from parquet
- connect to sqlite
- connect to postgres

### CODE REVIEW

File: examples/sql/connect_to_postgres.py [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+        ## Copy data from Postgres to duckdb
+
+        To prevent duckdb from continuously re-reading tables from PostgresSQL, you can copy the PostgresSQL databases into DuckDB. Note that this will consume your system's RAM.
+        """
+    )
+    return
+
+
+@app.cell
+def __(db, duckdb_table, mo, test_table):
```
Comment (mscolnick) [inline examples/sql/connect_to_postgres.py:150]: unrelated, but this import `duckdb_table` seems incorrect.

File: examples/sql/read_json.py [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        This notebook shows how to read a JSON file from a local file or a URL into an in-memory table.
+        """
+    )
+    return
+
+
+@app.cell(hide_code=True)
+def __():
+    import marimo as mo
+    import pandas as pd
```
Comment (mscolnick) [inline examples/sql/read_json.py:33]: super nit; but i feel like we should use polars (instead of pandas) when possible

Comment (akshayka) [reply examples/sql/read_json.py:33]: yea i agree in general but i'd like to host these on WASM too

File: examples/sql/connect_to_postgres.py (hunk 1 above)
Comment (akshayka) [reply examples/sql/connect_to_postgres.py:150]: yea weird that it's a def and a ref ...

Comment (akshayka) [reply examples/sql/connect_to_postgres.py:150]: The Python code has logic for not registering something as a ref if it's a def, SQL could be missing that

File: examples/sql/read_json.py (hunk 2 above)
Comment (mscolnick) [reply examples/sql/read_json.py:33]: oh yes good call - we did discuss that

**Review by mscolnick — APPROVED** [review]

**Pull request merged**